### DISCUSSION TITLE
Can't install lazer, nothing in event viewer

### DISCUSSION THREAD

**Discussion by dialupmars** [ppy/osu#17692] [Q&A] [answered]
when i click install.exe, it fails with this error code 
![Screenshot_43](https://user-images.githubusercontent.com/103147073/162065748-82f4f59e-96ff-4291-8106-9a5193d93426.png)
i checked for the osu folder in %appdata%, but there isn't one. I have normal osu installed and logs there don't change either.
Checked event viewer, can't see anything about the install ever trying to happen.

**Comment by smoogipoo** [comment]
https://github.com/ppy/osu/discussions/17649

**Reply by dialupmars** [reply]
that guy ended up being able to install it, i still cannot

**Comment by dialupmars** [comment, marked as answer]
Nevermind, turns out i was extremely low on disk space for my C: Drive and that was causing the install to fail.

**Reply by caesay** [reply]
Would you be able to check `%localappdata%\SquirrelClowdTemp\Squirrel.log` and share the logs for the installer? I would like to improve the error dialogs so issues like this are more obvious to the user.

**Reply by dialupmars** [reply]
@caesay hey there, sorry for the long reply time, here is the Squirrel.log
[Squirrel.log](https://github.com/ppy/osu/files/9199052/Squirrel.log)
